docs-client/src/containers/App/index.tsx

```diff
@@ -309,13 +322,20 @@ class App extends React.PureComponent<Props, State> {
     mobileDrawerOpen: false,
     specification: undefined,
     servicesOpen: true,
+    serviceOpens: {},
     enumsOpen: true,
     structsOpen: true,
     exceptionsOpen: true,
   };
 
   public componentWillMount() {
-    this.fetchSpecification();
+    this.fetchSpecification().then((specification) => {
+      let serviceOpens = {};
```

Comment:
Would it be simpler to do this inside `fetchSpecification` which is an async method?

Reply:
Yes, it would.
Let me move the codes inside `fetchSpecification`.
I just thought that I had to change the name `fetchSpecification` to something else because it does one more job inside the method and I couldn't figure out the better name. :)